Remotion/ObjectBinding/Web/Services/BusinessObjectWithIdentityProxy.cs

```diff
@@ -46,13 +47,15 @@ public BusinessObjectWithIdentityProxy (IBusinessObjectWithIdentity obj)
       _displayName = obj.GetAccessibleDisplayName();
     }
 
-    public string UniqueIdentifier
+    [DisallowNull]
+    public string? UniqueIdentifier
     {
       get { return _uniqueIdentifier; }
       set { _uniqueIdentifier = ArgumentUtility.CheckNotNullOrEmpty ("value", value); }
     }
```

Comment:
Cool null-"bug". It's obviously written this way, and it works, it's just...stupid, I guess?

TODO-Ticket to make this always nullable (after thinking about whether this is what we want)